clients/ts/signalr/src/NodeHttpClient.ts

```diff
@@ -33,7 +34,9 @@ export class NodeHttpClient extends HttpClient {
                 port: url.port,
             };
 
-            const req = http.request(options, (res: http.IncomingMessage) => {
+            const httpOrHttps: any = url.protocol === "https" ? https : http;
```

Comment:
`const httpOrHttps: typeof http | typeof https;` doesn't work I believe because it can't correctly resolve `.request` below since one returns `ClientRequest` and the other returns `http.ClientRequest` even though they are the same thing.

Reply:
Stick that comment in the code, it'll last longer ;P